.github/workflows/package-cloud-download-schedule.yml

```diff
@@ -9,11 +9,7 @@ env:
   PACKAGE_CLOUD_ADMIN_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_ADMIN_API_TOKEN }}
 on:
   schedule:
-    - cron: "0 16 * * *"
-
-  push:
-    branches:
-      - "**"
```

Comment:
I do not want statistics packagecloud download pipelines to execute anymore since it takes too much time nd too much resource. I don't make any changes there anymore

Reply:
It will still run as a cronjob run but will not get triggered on each branch push, right ? If that's is the case, I'm ok.

Reply:
But even if that's the case, that could be done in another PR since it's hard to relate it to enterprise changes.

Reply:
Yes it's that case. Schedule is unaffected .Everybody is so busy so I wanted to handle and make it approved in this PR
I will be more careful for further PR's